online_pmi

```diff
@@ -2,21 +2,23 @@
 
 """Align words based on stepwise EM alignments with PMI scores."""
 
-from collections import defaultdict
+from collections import defaultdict, Counter
 import itertools as it
 import sys
 import igraph, utils
```

Comment:
We need to fix imports in this file. `utils` for example is not needed in this file and we also need to use correct import paths. `utils` is imported as `infomapcog.utils`